### ISSUE TITLE
Suggestion feature: Add kv language support within Python files

### ISSUE THREAD

**Issue by Thong-Tran** [Monk3yDev/kvlang-vscode#1]
Can you add this to you code:
https://github.com/BattleBas/kivy-vscode/commit/c03039c300ebebaea0c6d5418833de8172f9c9a7#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

This is not mine but I think this is a good feature for your extension.

Thank for your extension. 👍

**Comment by Monk3yDev** [comment]
Thank you for the feature suggestion.  Basic support(grammars, snippets, syntaxes) of embedded language in python files has been added with one modification. To activate new functionality special keyword must be used for language detection. 

```
#<KvLang>
#</KvLang>
```

![kvlang_embedded_language](https://user-images.githubusercontent.com/8431461/54494776-358ec780-48de-11e9-84ea-e6cb11f5eefa.png)

### Why special keyword instead of Builder.load_string?
* It is generic syntax,
* String can be assigned to variable and later variable is used in Builder.load_string,
* Catch KvLang inside triple quotes,
* Python syntax will be only overridden between special keywords. 

### Additional information
Currently language server do not support  embedded language

**Comment by Thong-Tran** [comment]
hm... If you want to active kvlang like this, we need a snippet because it's quite long.

Can your extension automatically add snippet to python?

**Comment by Monk3yDev** [comment]
Here is example of 2 snippets which can be added to the extension. What do you think about this solution?

![kvlang_snippet_embedded_builder](https://user-images.githubusercontent.com/8431461/54561395-d8217600-49c4-11e9-8cd2-bf1539943e98.png)
![kvlang_snippet_embedded_variable](https://user-images.githubusercontent.com/8431461/54561396-d952a300-49c4-11e9-93ae-d25acb4c38d1.png)

Unfortunately snippets are always adjusted to the indent of the line it is inserted at. Manual action is needed to remove whitespaces from the string. 

Expected:
```python
 →  →  →  → Builder.load_string('''
#<KvLang>
Content
#</KvLang>
''')
```
Actual:
```python
           Builder.load_string('''
→  →  →  → #<KvLang>
→  →  →  → Content
→  →  →  → #</KvLang>
→  →  →  → ''')
```
Here is open issue in vscode for indentation [https://github.com/Microsoft/vscode/issues/20112](url).

**Comment by Thong-Tran** [comment]
Oh, It's good 👍. Thank for your work 😁

P/s: Also about [that problem](https://github.com/Microsoft/vscode/issues/20112), I think it's always written at the beginning of the code so it won't be a big problem for this case.

**Comment by Monk3yDev** [comment]
Basic support was released in version 0.0.5 of KvLang. If you have more suggestions or improvements please open new issue.
